babel.config.json

```diff
@@ -2,8 +2,6 @@
     "presets": ["@babel/preset-env", "@babel/preset-react"],
     "plugins": [
         ["@babel/plugin-proposal-decorators", {"legacy": true}],
-        "@babel/plugin-proposal-object-rest-spread",
```

Comment:
this is included in the `@babel/preset-env`: https://babeljs.io/docs/en/babel-plugin-proposal-object-rest-spread